server/server.mjs

```diff
@@ -30,21 +30,32 @@ const checkJwt = jwt({
   algorithms: ["RS256"],
 });
 
-const tasks = express.Router();
+const user = express.Router();
 
-tasks.get("/", async (request, response) => {
-  const tasks = await db.getTasks();
-  response.json(tasks);
+user.use(express.json());
+
+user.get("/:email", async (request, response) => {
+  let dbUser = await db.getUser(request.params.email);
+
+  if (!dbUser) {
+    //user does not exist, so create user with just the email
+    dbUser = db.createUser(request.params.email);
+  }
+  const responseUser = {
+    userName: dbUser.username,
+    phoneNumber: dbUser.phone,
+    zipCode: dbUser.zipcode,
```

Comment:
If you want to you can validate the `dbUser.zipcode` to make sure that it is a 5-digit number before you send it to where it breaks.